src/main/java/sc/fiji/snt/annotation/AllenCompartment.java

```diff
@@ -340,7 +336,7 @@ public OBJMesh getMesh() {
 	 */
 	@Override
 	public boolean isParentOf(final BrainAnnotation childCompartment) {
-		if (childCompartment == null || !(childCompartment instanceof AllenCompartment))
```

Comment:
@carshadi , won't you get a NPE here  if the childCompartment being tested is null? nodes in a reconstruction can be associated with a null compartment

Reply:
IntelliJ suggested that the null check was not necessary, apparently `instanceof` returns `false` if the operand is null
https://stackoverflow.com/questions/2950319/is-null-check-needed-before-calling-instanceof

Reply:
I did not know that! good to know. Oh boy. I should adopt IntelliJ. I have never be warned in Eclipse, and have been doing null checks from day one!